pkg/hostpath/hostpath.go

```diff
@@ -102,13 +107,10 @@ func NewHostPathDriver(driverName, nodeID, endpoint string, ephemeral bool) (*ho
 func (hp *hostPath) Run() {
 	s := NewNonBlockingGRPCServer()
 
-	hp.ids = nil
-	hp.cs = nil
+	hp.ids = NewIdentityServer(hp.name, hp.version)
```

Comment:
Does Kubernetes require an identity server? I thought they were optional in CSI.

Reply:
@kfox1111 Yes, the Registrar sidecar calls Identity services to retrieve information about the driver.